content/devices/onclite.md

```diff
@@ -0,0 +1,99 @@
+---
+draft: false
+title: Xiaomi Redmi 7 (onclite)
+---
+> **Make a backup now, as your device will be wiped.**
+## Before you proceed
```

Comment:
Please add a template file instead, Please refer readme file for updated instructions.